src/main/java/frc/robot/Robot.java

```diff
@@ -74,6 +74,9 @@ public void disabledPeriodic() {
    */
   @Override
   public void autonomousInit() {
+
+    m_robotContainer.setupAuto();
+
```

Comment:
Let's also move the autonomous command logic (line 80-85) into our `setupAuto()`, and remove `RobotContainer::getAutonomousCommand()`. Might as well keep all our autonomous code together.

Reply:
Resolved in 0bf8ea1